<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDialog</class>
 <widget class="QDialog" name="MyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEditFirstNum">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSecondNum">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFirstNum">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>First number (-9 to 9)</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSecondNum">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Second number (-9 to 9)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxTheme">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dark/Light theme</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTimer">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimer">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time (0 to 30 sec)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDialogConfirm">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
